save.go

```diff
@@ -15,6 +15,7 @@ import (
 func saveImage(image imgutil.Image, additionalNames []string, logger Logger) (platform.ImageReport, error) {
 	var saveErr error
 	imageReport := platform.ImageReport{}
+	logger.Infof("Saving %s\n", image.Name())
```

Comment:
```suggestion
	logger.Infof("Saving %s...\n", image.Name())
```

Reply:
IDK if this is really needed it is just a preference. Having tried a build with this branch out, I must say the message when there's a pause is psychologically satisfying :)

Reply:
Actually there's no `...` on the other "Doing XXX" log messages so what we have now is probably more consistent.

Reply:
@jabrown85 I'm sorry... I made this comment and then tried to take it back :/ I am totally fine with whatever we decide to do :)

Reply:
I liked the `...` suggestion. But if someone feels strongly that we shouldn't introduce that, I'll back it out.